### PR TITLE
chore: adds worker service manifest

### PR DESCRIPTION
**Pull request by xar-tol**
Adds file for new worker service manifest.

Addresses #2550

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: templates/workloads/services/worker/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+#secrets:                      # Pass secrets from AWS Systems Manager (SSM) Parameter Store.
+#  GITHUB_TOKEN: GITHUB_TOKEN  # The key is the name of the environment variable, the value is the name of the SSM parameter.
+
+# You can override any of the values defined above by environment.
+#environments:
+#  test:
+#    count: 2               # Number of tasks to run for the "test" environment.
+
+# You can enable self-publishing or publishing to other services.
```
Comment (bvtujo) [inline templates/workloads/services/worker/manifest.yml:55]: :heart:

File: templates/workloads/services/worker/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# The manifest for the "{{.Name}}" service.
```
Comment (bvtujo) [inline templates/workloads/services/worker/manifest.yml:1]: Can we add 
```yaml
# Read the full specification for the "Worker Service" type at:
#  https://aws.github.io/copilot-cli/docs/manifest/worker-service/
```

**Review by bvtujo — APPROVED** [review]
This looks great, just a tiny suggestion! Feel free to remove the label when it's addressed.

File: templates/workloads/services/worker/manifest.yml (hunk 2 above)
Comment (xar-tol) [reply templates/workloads/services/worker/manifest.yml:1]: Adjusted to add the URL and statement.

**Review by iamhopaul123 — APPROVED** [review]
LGTM :shipit:

File: templates/workloads/services/worker/manifest.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+memory: {{.Memory}}    # Amount of memory in MiB used by the task.
+count: {{.Count.Value}}       # Number of tasks that should be running in your service.
+exec: true     # Enable running commands in your container.
+
+{{- if .Subscribe}}{{- if .Subscribe.Topics}}
+subscribe:
+  topics:
+{{- range $topic := .Subscribe.Topics}}
+    - name: {{.Name}}
+      service: {{.Service}}
```
Comment (iamhopaul123) [inline templates/workloads/services/worker/manifest.yml:38]: This might not work....

Comment (xar-tol) [reply templates/workloads/services/worker/manifest.yml:38]: Would it need an additional line break like the [pipelines](https://aws.github.io/copilot-cli/docs/manifest/pipeline/)?

Comment (xar-tol) [reply templates/workloads/services/worker/manifest.yml:38]: My plan is that the user will be able to pick from any available topics in init, so those can be used to fill the manifest after init here if any are chosen.

Comment (iamhopaul123) [reply templates/workloads/services/worker/manifest.yml:38]: Yeah but should we use `$topic`? Otherwise `.Name` might just be the service name.

Comment (xar-tol) [reply templates/workloads/services/worker/manifest.yml:38]: Good point - I can toss this adjustment to $topic.Name into the worker service manifest ds PR up now.

**Pull request merged**